msys2-runtime/PKGBUILD

```diff
@@ -202,13 +204,14 @@ prepare() {
   git am --committer-date-is-author-date "${srcdir}"/0047-Make-paths-WCS-MBS-conversion-explicit.patch
   git am --committer-date-is-author-date "${srcdir}"/0048-Handle-ORIGINAL_PATH-just-like-PATH.patch
   git am --committer-date-is-author-date "${srcdir}"/0049-kill-kill-Win32-processes-more-gently.patch
-  git am --committer-date-is-author-date "${srcdir}"/0050-Generate-a-.pdb-file-instead-of-a-.dbg-file.patch
-  git am --committer-date-is-author-date "${srcdir}"/0051-Fix-incorrect-path-conversion-trailing-slash.patch
-  git am --committer-date-is-author-date "${srcdir}"/0052-Add-a-helper-to-obtain-a-function-s-address-in-kerne.patch
-  git am --committer-date-is-author-date "${srcdir}"/0053-squash-Try-to-handle-Ctrl-C-on-Win32-processes-the-s.patch
-  git am --committer-date-is-author-date "${srcdir}"/0054-fixup-kill-kill-Win32-processes-more-gently.patch
-  git am --committer-date-is-author-date "${srcdir}"/0055-srandomdev-avoid-warning-about-uninitialized-use.patch
-  git am --committer-date-is-author-date "${srcdir}"/0056-squash-Fix-incorrect-path-conversion-trailing-slash.patch
+  git am --committer-date-is-author-date "${srcdir}"/0050-Fix-incorrect-path-conversion-trailing-slash.patch
+  git am --committer-date-is-author-date "${srcdir}"/0051-Add-a-helper-to-obtain-a-function-s-address-in-kerne.patch
+  git am --committer-date-is-author-date "${srcdir}"/0052-squash-Try-to-handle-Ctrl-C-on-Win32-processes-the-s.patch
+  git am --committer-date-is-author-date "${srcdir}"/0053-fixup-kill-kill-Win32-processes-more-gently.patch
+  git am --committer-date-is-author-date "${srcdir}"/0054-srandomdev-avoid-warning-about-uninitialized-use.patch
+  git am --committer-date-is-author-date "${srcdir}"/0055-squash-Fix-incorrect-path-conversion-trailing-slash.patch
+  git am --committer-date-is-author-date "${srcdir}"/0056-Generate-a-.pdb-file-instead-of-a-.dbg-file.patch
+  git am --committer-date-is-author-date "${srcdir}"/0057-Split-.pdb-generation-into-separate-files.patch
```

Comment:
Moved the original PDB generation commit to the end, so that it comes just before my patch.